### DISCUSSION TITLE
Help! Macbook

### DISCUSSION THREAD

**Discussion by SmokeeakaRoxy** [MobiFlight/MobiFlight-Connector#1760] [Q&A]
Hey yall i got a early 2015 macbook pro and i wanna run mobiflight on it but i only see a windows version.

Is there a version for mac?

**Comment by tigert** [comment]
MobiFlight is windows only, so unfortunately not. 

This discussions tab in github is really used more for development planning, so you are better off to join the Discord server to get support for using MobiFlight.
